pnparser/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .pnparser import PatchNotesParser
```

Comment:
can you call it patchnoteparser? no one will ever need to type the full name of the cog and to an outsider `pnparser` is pretty meaningless